torch/csrc/jit/custom_graph_executor_impl.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <torch/csrc/jit/graph_executor_impl.h>
+
+#include <ATen/core/interned_strings.h>
+
+namespace torch {
+namespace jit {
+
+using GraphExecutorImplCreator =
+    std::function<GraphExecutorImplBase*(std::shared_ptr<Graph>, bool)>;
+
+static const Symbol kDefaultExecutor =
+    Symbol::fromQualString("executor::default");
+static const Symbol kProfilingExecutor =
```

Comment:
Well this puts a `static` symbol into every compliation unit that includes this part, so it's a bit wasteful. Either declare it extern, or just use `fromQualString` in every place that needs this.